### DISCUSSION TITLE
Something does not work

### DISCUSSION THREAD

**Discussion by paoloparonuzzi** [Argonne-National-Laboratory/DSP#221] [Q&A]
Hello,

I am trying to use DSP to solve a problem with a block structure formulation. I only need a "Serial Run".
The problem contains integer variables both in the (master) coupling constraint and in the subproblems. So, if I well understood, algorithm "dw" is the one I should/can use to solve my problem, as branch and bound is needed.
Since I am not familiar with SMPS format, I would prefer to use MPS+DEC format. I wrote the model using this format and I tried to solve it using "--algo dw" option.
If I try to run the example:
        ./bin/runDsp --algo dw \
                     --mps ../examples/mps-dec/noswot.mps \
                     --dec ../examples/mps-dec/noswot.dec \
                     --soln mysoln \
                     --param myparam.txt
it seems that everything correctly works and a solution with value -41 is found.
If I try to solve the instance of my problem, then a not-optimal solution of value 1e+75 is found. I do not understand why.
May you give me some hints to find the mistake?
I am attacching MPS, DEC and output files
[DSP.zip](https://github.com/Argonne-National-Laboratory/DSP/files/8410789/DSP.zip)

Of course, I can provide you any other file or info that can help in detecting the problem.
Many thanks in advance.

**Comment by kibaekkim** [comment]
@paoloparonuzzi Sorry for the late response. I was not watching the discussion tab. Has this been resolved? I remember I fixed some bugs in `dw` around that time. But I am not sure if that resolves your issue.
